cg/server/api.py

```diff
@@ -220,53 +220,56 @@ def samples():
     if request.args.get("status") and not g.current_user.is_admin:
         return abort(http.HTTPStatus.FORBIDDEN)
     if request.args.get("status") == "incoming":
-        samples_q = db.samples_to_receive()
+        samples_q: List[Sample] = db.get_all_samples_to_receive()
     elif request.args.get("status") == "labprep":
-        samples_q = db.samples_to_prepare()
+        samples_q: List[Sample] = db.get_all_samples_to_prepare()
     elif request.args.get("status") == "sequencing":
-        samples_q = db.samples_to_sequence()
+        samples_q: List[Sample] = db.get_all_samples_to_sequence()
     else:
         customer_objs: Optional[Customer] = (
             None if g.current_user.is_admin else g.current_user.customers
         )
-        samples_q = db.samples(enquiry=request.args.get("enquiry"), customers=customer_objs)
+        samples_q: List[Sample] = db.get_samples_by_enquiry(
+            enquiry=request.args.get("enquiry"), customers=customer_objs
+        )
     limit = int(request.args.get("limit", 50))
-    data = [sample_obj.to_dict() for sample_obj in samples_q.limit(limit)]
-    return jsonify(samples=data, total=samples_q.count())
+    data = [sample_obj.to_dict() for sample_obj in samples_q[:limit]]
```

Comment:
Maybe a type hint here
```suggestion
    data: List[Dict] = [sample_obj.to_dict() for sample_obj in samples_q[:limit]]
```